lib/widgets/theme.dart

```diff
@@ -136,6 +136,9 @@ class DesignVariables extends ThemeExtension<DesignVariables> {
       title: const Color(0xff1a1a1a),
       streamColorSwatches: StreamColorSwatches.light,
       star: const HSLColor.fromAHSL(0.5, 47, 1, 0.41).toColor(),
+      errorBannerLabel: const Color.fromRGBO(133, 42, 35, 1),
+      errorBannerBackground: const Color.fromRGBO(238, 222, 221, 1),
+      errorBannerBorder: const Color.fromRGBO(132, 41, 36, 0.4),
```

Comment:
Since the web css uses HSL, let's use `HSLColor.fromAHSL(…).toColor()`, so it's easier to check against web. (Here and in `.dark` below.)